gradebookng/tool/src/java/org/sakaiproject/gradebookng/GradebookNgApplication.java

```diff
@@ -75,14 +80,18 @@ public void init() {
 		// getApplicationSettings().setPageExpiredErrorPage(getHomePage());
 
 		// show internal error page rather than default developer page
-		// for production, set to SHOW_NO_EXCEPTION_PAGE
-		getExceptionSettings().setUnexpectedExceptionDisplay(IExceptionSettings.SHOW_EXCEPTION_PAGE);
+		boolean showStackTraces = ServerConfigurationService.getBoolean(SAK_PROP_PORTAL_SHOW_ERROR, SAK_PROP_PORTAL_SHOW_ERROR_DEFAULT);
```

Comment:
should be injected properly rather than static.

Reply:
Actually this can all be moved to the errorpage

Reply:
Thanks, @steveswinsburg. Is using Spring injection required for SCS? The cover is not deprecated and I don't think there are plans on deprecating/removing it. Since it's only being used once, and in one spot, I thought it was better to just use the cover.